src/components/member-profile/contribution-type/index.js

```diff
@@ -54,7 +65,22 @@ const ContributionType = (props) => {
       <div className={showMoreContentClass}>
         {type !== 'Active tasks' ? (
           <div>
```

Comment:
It would make it easier to read & manage later if we make this out as a component.